assets/src/blocks/index-frontend.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * @todo when there is javascript/css for blocks that needs to be on the
+ * frontend, this is where you import/export it from (final script/styles
+ * built from this file
+ */
```

Comment:
This is just  a placeholder for now.  But this is basically the entry script for the build process so it can be used to import any common script/style modules between admin and frontend and would then be built for consumption on the frontend.